docs/UserGuide.md

```diff
@@ -524,8 +546,106 @@ To check that your system has the correct Java version (Java 11 and above) to ru
+
+### 8.3 My data file is corrupt
+
+Do not launch the app if you detect this issue, since the `myinsurec.json` file will be cleared of all data.
+To recover:
+
+1. [create a backup](#553-creating-and-using-data-file-backups) of the file.
+2. Edit all invalid fields and format issues manually in the backup file.
+    * What should I look out for while editing? :
+    * Check that the structure of the file is correct (no missing semicolons, commas, brackets, and data fields). <br> It helps to be familiar with the [JSON file](#json-file) format here, but most users can quickly pick up the basics by referring to a working copy of the data file.
+    * Some invalid data values to look out for :
+    * invalid dates for client birthday, or meeting date e.g. 00000000 (invalid DDMMYYYY), 1212202020 (too many digits),
+    * Invalid meeting start time or end time e.g. 2500 (invalid HHMM).
+3. [Use this edited file](#553-creating-and-using-data-file-backups) when launching *MyInsuRec*.
+
 --------------------------------------------------------------------------------------------------------------------
 
-## 9. Glossary
+## 9. Acknowledgements
+
+This project is based on the [AB3 project template](https://github.com/se-edu/addressbook-level3) by [se-education.org](https://se-education.org/).
+
+We thank the CS2103T and CS2101 teaching team and all our classmates for supporting us in this project!
+
+## 10. Glossary
+
+### Quick Reference
+
+- [CLI](#cli)
+- [GUI](#gui)
+- [INDEX](#)
+- [JSON file](#json-file)
+- [JAR file](#jar-file)
+- [Parameter](#parameter)
+
+### *C*
+
+##### CLI 
+
+Command Line Interface, user interface that accepts input as lines of text.
+
+### *G*
+
+##### GUI
+
+Graphical User Interface, user interface that accepts input in means other than text, such as mouse clicks.
+
+### *I*
+
+##### INDEX
+
+Number indicating the position of a client, meeting, or product in their respective lists.
+
+In all commands `INDEX` **must be a positive integer** 1, 2, 3, …​`
+
+In `editClient`, `delClient`, and `viewClient` commands, `INDEX` refers to the  number shown by executing [`listClient`](#512-list-clients-listclient) command.
+
+e.g. David Li has position of 4 in the shown [client list](#512-list-clients-listclient)
+![](images/listClient.png)
+
+
+In `viewMeeting`, `delMeeting`, and `editMeeting` commands, `INDEX` refers to the number shown by executing [`listMeeting`](#522-list-meetings-listmeeting) command.
+
+e.g. The meeting shown has position of 1 in [meeting list](#522-list-meetings-listmeeting)
+![](images/listMeeting.png)
+
+In `delProduct` command, `INDEX` refers to the number shown by executing [`listProduct`](#532-list-products-listproduct) command. 
+
+e.g. Product6 has position of 5 in the shown [product list](#332-listing-products-listproduct):
+![](images/listProduct.png)
+
+### *J*
+
+##### JSON file
+
+A file that stores data that has been structured according to the JSON data format.
```

Comment:
Could proabably provide a link to some online resources to know more about json format

Reply:
ahh good idea, will add that in